doc/api/traceroute/error.md

```diff
@@ -0,0 +1,20 @@
+# NAME
+Error -- A traceroute error.
+
+# LIBRARY
+MeasurementKit (libmeasurement_kit, -lmeasurement_kit).
+
+# SYNOPSIS
+```C++
+#include <measurement_kit/traceroute.hpp>
+
+mk::Error error = mk::traceroute::EOFError();
```

Comment:
Here I'd list traceroute errors along with a terse explanation (see for example the doc of DNS errors).